main/tests.py

```diff
@@ -37,8 +21,8 @@ def test_enter_logout_page(self, client):
         response = client.get('/logout/')
         assert response.status_code == 200
 
-    def test_successful_login_and_logout(self, client, make_user):
-        sample_user = make_user("test1@gmail.com", "testy", "test", "1995-05-05", "test123")
+    def test_successful_login_and_logout(self, client):
+        sample_user = get_user_model().objects.create_user("test1@gmail.com", "testy", "test", "1995-05-05", "test123")
```

Comment:
You're creating the same user in all the tests here. Won't that cause a problem?
You should probably use a setup function and create the user once

Reply:
`make_user` was a fixture that did exactly what `get_user_model().objects.create_user` is doing, so I changed it